src/box/lua/xlog.c

```diff
@@ -365,8 +365,48 @@ lbox_xlog_parser_open_pairs(struct lua_State *L)
 	return 3;
 }
 
+/** Opens an xlog file and returns its meta block. */
+static int
+lbox_xlog_parser_open_meta(struct lua_State *L)
+{
+	int args_n = lua_gettop(L);
+	if (args_n != 1 || !lua_isstring(L, 1))
+		luaL_error(L, "Usage: parser.open(log_filename)");
```

Comment:
Nit: the documentation request title states the argument as optional, but it isn't.

Reply:
Fixed